payment_paypal/indico_payment_paypal/plugin.py

```diff
@@ -76,3 +92,58 @@ def adjust_payment_form_data(self, data):
         data['return_url'] = url_for_plugin('payment_paypal.success', registration.locator.uuid, _external=True)
         data['cancel_url'] = url_for_plugin('payment_paypal.cancel', registration.locator.uuid, _external=True)
         data['notify_url'] = url_for_plugin('payment_paypal.notify', registration.locator.uuid, _external=True)
+
+# PayPal Itemized cart get all billable items
+        itemized_data = []
+        if registration.base_price:
+            self.create_choice_data(itemized_data, 
+                               'Registration Fee', 
```

Comment:
I think this should look more like this

```
self.create_choice_data(itemized_data, 'Registration Fee', '', registration.base_price, 1, registration.base_price_info)
```

or if this doesn't fit in one line

```
self.create_choice_data(itemized_data, 'Registration Fee', '', registration.base_price,
                        1, registration.base_price_info)
```